x/mongo/driver/description/server.go

```diff
@@ -246,6 +247,35 @@ func NewServer(addr address.Address, response bsoncore.Document) Server {
 				return desc
 			}
 			desc.Tags = tag.NewTagSetFromMap(m)
+		case "topologyVersion":
+			doc, ok := element.Value().DocumentOK()
+			if !ok {
+				desc.LastError = fmt.Errorf("expected 'topologyVersion' to be an document but it's a BSON %s", element.Value().Type)
+				return desc
+			}
+			elements, err := doc.Elements()
+			if err != nil {
+				desc.LastError = err
+				return desc
+			}
+			var tv TopologyVersion
+			for _, element := range elements {
+				switch element.Key() {
+				case "processId":
+					tv.ProcessID, ok = element.Value().ObjectIDOK()
+					if !ok {
+						desc.LastError = fmt.Errorf("expected 'processId' to be a objectID but it's a BSON %s", element.Value().Type)
+						return desc
+					}
+				case "counter":
+					tv.Counter, ok = element.Value().AsInt64OK()
```

Comment:
Can we call `Int64OK` instead? The spec PR specifically lists the `counter` field as int64 so I think it would actually be a server bug if they replaced this with a smaller integer or double instead.